### PR TITLE
Document altenative exports

### PR DESCRIPTION
**Pull request by Aquassaut**
This contains the doc for alternative exports. Story [here](https://app.clubhouse.io/opendatasoft/story/16350/document-alternative-export-mgmt-api-on-slate). I also noticed some issues with the attachements doc that I have fixed here since both docs are linked in so many ways.

### CODE REVIEW

**Comment by Aquassaut** [comment]
[ch16350]

File: source/index.html.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ includes:
   - dataset_resources
   - dataset_metadata
   - dataset_attachments
+  - dataset_attachments
```
Comment (juliendbg) [inline source/index.html.md]: It should be `dataset_alternative_exports` no?

**Review by juliendbg — APPROVED** [review]
All good except the typo in the index.

File: source/index.html.md (hunk 1 above)
Comment (Aquassaut) [reply source/index.html.md]: It sure should

**Pull request merged**